docs/dev/file_encryption.md

```diff
@@ -0,0 +1,76 @@
+File level encryption in scylla enterprise
```

Comment:
nit, might want to use "docs" as the prefix of the commit message's title.

Reply:
done